recipe/meta.yaml

```
@@ -1,13 +1,13 @@
 {% set name = "solvation_analysis" %}
-{% set version = "0.1.5" %}
+{% set version = "0.3.1" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/solvation_analysis-{{ version }}.tar.gz
```

```suggestion
  url: https://pypi.io/packages/source/{{ name[0] }}/solvation-analysis/{{ name }}-{{ version }}.tar.gz
```
You need a standard hyphen for the first part of this shortened url.
See: https://github.com/pypi/warehouse/issues/13240